<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMockGen</class>
 <widget class="QMainWindow" name="GMockGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>362</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GMockGen</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browserfile">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>29</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browserfolder">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>69</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutputUrl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>255</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>254</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Location:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browser">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>252</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>691</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mockbutton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>280</y>
      <width>112</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cppcheck">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>225</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Files</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="hppcheck">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>225</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Header Files</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelbutton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress: 0 / 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>451</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyright by LG - hung7.nguyen@lge.com, trang3.le@lge.com</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
